x-pack/plugins/ml/public/alerting/config_validator.tsx

```diff
@@ -66,9 +61,9 @@ export const ConfigValidator: FC<ConfigValidatorProps> = React.memo(
               <li>
                 <FormattedMessage
                   id="xpack.ml.alertConditionValidation.alertIntervalTooHighMessage"
-                  defaultMessage="The check interval is greater than the maximum bucket span of the selected jobs. Reduce it to {resultBucketSpan} to avoid excessive delay in receiving notifications."
+                  defaultMessage="The check interval is greater than the lookback interval. Reduce it to {lookbackInterval} to avoid missing potential notifications."
```

Comment:
```suggestion
                  defaultMessage="The check interval is greater than the lookback interval. Reduce it to {lookbackInterval} to avoid potentially missing notifications."
```

Reply:
Updated in https://github.com/elastic/kibana/pull/97370/commits/0a445ded6eb7e70e319e9cc9a414ccaea657df37